### PR TITLE
Document server hardware

### PR DESCRIPTION
**Pull request by abitrolly**

### CODE REVIEW

**Comment by abitrolly** [comment]
The OSM org site uses Chef to autogenerate https://hardware.openstreetmap.org/

And https://github.com/openstreetmap/operations is the tracker for coordination of all hardware tasks .

**Review by tbicr — COMMENTED** [review]
lets move it to readme

**Comment by abitrolly** [comment]
Moved to `README.md`

File: index.py [hunk 1]
```diff
@@ -679,7 +679,27 @@ def deg2num(lat_deg, lon_deg, zoom):
...
+                path = environ.get("PATH_INFO", "")
+                relpath = path[1:]  # strip leading "/" if exists
+                if os.path.isfile(relpath):
+                    return web.httpserver.StaticApp(environ, start_response)
+                else:
+                    return app(environ, start_response)
+
+        return RootMiddleware()
+
+    app.run(root_static)
```
Comment (tbicr) [inline index.py:702]: this part should works fine in master, anyway I prefer to focus on documentation only described in PR

**Review by tbicr — CHANGES_REQUESTED** [review]

File: index.py (hunk 1 above)
Comment (abitrolly) [reply index.py:702]: Yep, I've rebased on top of my master, which was not clean, apparently. That will take need some time for the back surgery.

**Review by tbicr — APPROVED** [review]
LGTM

**Pull request merged**